documentation/modules/ROOT/pages/connectors/oracle.adoc

```diff
@@ -1910,6 +1910,33 @@ This also enables tracking DDL changes against captured tables, so if the schema
 This allows LogMiner to mine substantially faster but at the expense that DDL changes cannot be tracked.
 If the captured table(s) schema changes infrequently or never, this is the ideal choice.
 
+|[[oracle-property-log-mining-buffer-type]]<<oracle-property-log-mining-buffer-type, `+log.mining.buffer.type+`>>
+|`memory`
+|The buffer type controls how the connector will manage buffering transaction data. +
+ +
+`memory` - Uses the JVM process' heap to buffer all transaction data.
+This is the ideal choice if you don't expect a lot of long-running or large transactions. +
```

Comment:
Choose this option if you don't expect the connector to process a high number of long-running or large transactions. +